routers/retraining_pipeline.py

```
@@ -0,0 +1,212 @@
+        )
+
+        return {
+            "success": True,
+            "triggered": True,
+            "task_id": task.id,
+            "message": "Retraining task queued. Poll GET /api/retraining/status?task_id= for progress.",
+        }
+
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)